Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp

```diff
@@ -42,6 +42,11 @@ void HybridPICModel::ReadParameters ()
 
     // convert electron temperature from eV to J
     m_elec_temp *= PhysConst::q_e;
+
+    // external currents
+    pp_hybrid.query("Jx_external_grid_function(x,y,z,t)", m_Jx_ext_grid_function);
```

Comment:
Could you add documentation for this on this page: https://warpx.readthedocs.io/en/latest/usage/parameters.html
In particular, could you state there that this is only used for the hybrid model?

Reply:
Thanks for the comment! I added it to the documentation.